examples/jenkins-integration/jenkinsfile

```diff
@@ -0,0 +1,175 @@
+                                sleep (6)
+                            }
+                        }
+                    }
+                }
+                stage ('Integ.') {
+                    stages {
+                        stage ('Run Integration Tests') {
+                            steps{
+                                echo ('UI Tests...')
+                                sleep (130)
+                            }
+                        }
+                        stage ('Send Integration Test Results') {
+                            steps{
+                                echo ('Send Integration Test Results...')
+                                sleep (4)
+                            }
+                        }
+                    }
+                }
+            }
+        }
+        
+         stage ('Bookmark Database') {
+            when {
+                equals expected: 'true', actual: test_error
+            }
+            steps{
+                script {
+                    echo ('Bookmark VDB... ')
+                    withCredentials([string(credentialsId: 'DCT_API_KEY', variable: 'KEY')]) {
+                        sh """
+                            curl -X 'POST' -k \
+                                'https://$DCT_HOSTNAME/v3/bookmarks' \
+                                -H 'accept: application/json' \
+                                -H 'Authorization: apk ${KEY}' \
+                                -H 'Content-Type: application/json' \
+                                -d '{
+                                    "name": "JKNS-BOOKMARK-$BUILD_NUMBER",
+                                    "vdb_ids": [
+                                        "${vdb_id}"
+                                    ],
+                                    "retain_forever": true,
+                                    "make_current_account_owner": true
+                                }'
+                        """
+                    }
+                }
+            }
+        }
+
+        stage ('Destroy Full Application') {
+            parallel {
+                stage ('Destroy Application') {
+                    steps {
+                        script {
+                            echo ('Destroying Application...')
+                            sleep (30)
+                        }
+                    }
+                }
+                stage ('Destroy Database w/ Terraform') {
+                    steps {
+                        script {
+                            if (provision_successful) {
+                                    sleep (60)
+                                    echo ('Destroying Test App and VDB...')
+                                    withCredentials([string(credentialsId: 'DCT_API_KEY', variable: 'KEY')]) {
+                                        sh ('terraform -chdir=simple-provision destroy -var="source_data_id_1=$SOURCE_VDB" -var="dct_hostname=$DCT_HOSTNAME" -var="dct_api_key=$KEY" -auto-approve')
```

Comment:
Update chdir value to if you bring in the full folder

`examples/simple-provision`

Reply:
done